node/src/chain_spec.rs

```diff
@@ -804,29 +837,77 @@ fn foucoco_genesis(
 			},
 		},
 		oracle: foucoco_runtime::OracleConfig {
-			max_delay: u32::MAX,
+			max_delay: 604_800_000,
 			oracle_keys: vec![
 				Key::ExchangeRate(CurrencyId::XCM(0)),
-				Key::ExchangeRate(TESTNET_DEFAULT_WRAPPED_CURRENCY_ID),
+				Key::ExchangeRate(CurrencyId::Native),
+				Key::ExchangeRate(TESTNET_STELLAR_NATIVE_CURRENCY_ID),
+				Key::ExchangeRate(TESTNET_USDC_CURRENCY_ID),
+				Key::ExchangeRate(TESTNET_BRL_CURRENCY_ID),
+				Key::ExchangeRate(TESTNET_TZS_CURRENCY_ID),
 			],
 		},
 		vault_registry: foucoco_runtime::VaultRegistryConfig {
-			minimum_collateral_vault: vec![(XCM(0), 0), (XCM(0), 0)],
-			punishment_delay: foucoco_runtime::DAYS,
-			secure_collateral_threshold: vec![(
-				default_pair(XCM(0)),
-				FixedU128::checked_from_rational(150, 100).unwrap(),
-			)],
-			/* 150% */
+			minimum_collateral_vault: vec![
+				(XCM(0), 3_000_000_000_000),
+				(XCM(0), 0), // Should we remove this repeated line?
+				(CurrencyId::Native, 0),
+				(TESTNET_STELLAR_NATIVE_CURRENCY_ID, 0),
+				(TESTNET_USDC_CURRENCY_ID, 0),
+				(TESTNET_BRL_CURRENCY_ID, 0),
+				(TESTNET_TZS_CURRENCY_ID, 0),
```

Comment:
In this vec we only specify the minimum collaterals for assets that we want to use as collateral. So we can remove all these Stellar assets from here as they will always be the wrapped asset and not the collateral asset.